Command/ChapleanApiLogCleanCommand.php

```diff
@@ -24,7 +40,12 @@ class ChapleanApiLogCleanCommand extends ContainerAwareCommand
     protected function configure()
     {
         $this->setName('chaplean:api-log:clean');
-        $this->setDescription('Delete logs older than one month');
+        $this->setDescription('Delete logs older than the mentionned date (1 month by default)');
+        $this->addArgument(
+            'minimumDate',
```

Comment:
- [ ] InputOption should not be in camelCase : --minimum-date (and eventually a shorter alias if wanted)

Reply:
It is an argument, not an option as the date is the core of the command.